cjsmin/src/chess.ts

```diff
@@ -1444,10 +1478,7 @@ export class Chess {
 
   loadPgn(
     pgnMoveLine: string,
-    {
-      strict = false,
-      newlineChar = '\r?\n',
```

Comment:
why get rid of` newlineChar = '\r?\n',`?

Reply:
unused